core/server/data/schema/schema.js

```diff
@@ -300,5 +300,13 @@ module.exports = {
         created_by: {type: 'string', maxlength: 24, nullable: false},
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    sessions: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        session_id: {type: 'string', maxlength: 32, nullable: false, unique: true},
+        user_id: {type: 'string', maxlength: 24, nullable: false},
+        session_data: {type: 'varchar', maxlength: 2000, nullable: false},
```

Comment:
`knex` uses `string`, because it's a generic query builder. It's a `varchar` in the db.